# FIXME: this is here because somehow pytest does not detect
#  local_node_eth being in the same file as local_node_eth_replay

from karpatkit.test_utils.fork import accounts, local_node_eth, local_node_gc
